quantum/process_keycode/process_unicode_common.c

```diff
@@ -16,6 +16,9 @@
 
 #include "process_unicode_common.h"
 #include "eeprom.h"
+#include <string.h>
+#include <ctype.h>
+#include <stdlib.h>
```

Comment:
Probably no need to include `<stdlib.h>` as it's already included by `quantum.h`, and definitions from it are used all over QMK without explicitly including it.